libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/processes-detail-item/dh-processes-detail-item.component.html

```diff
@@ -45,4 +48,12 @@
     <!--   class='open-in-new-icon'-->
     <!-- ></watt-icon>-->
   </div>
+  <div class="child-cell error-row" *ngIf="detail.errors.length > 0">
+    <watt-validation-message
+      *ngFor="let error of detail.errors"
+      [label]="'meteringPoint.overview.processesTab.processDenied' | transloco"
+      [message]="error.description"
+      [type]="errorValidationType"
```

Comment:
nitpick: `WattValidationMessageType` is a "type" and not an "enum", so we can write the desired value directly in the template (and get rid of the extra component property). I.e.

```suggestion
      type="danger"
```